service/rtc/sfu.go

```diff
@@ -83,10 +109,35 @@ func initInterceptors(m *webrtc.MediaEngine) (*interceptor.Registry, error) {
 
 	// RTCP Reports
 	if err := webrtc.ConfigureRTCPReports(&i); err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	return &i, nil
+	// TWCC
+	if err := webrtc.ConfigureTWCCSender(m, &i); err != nil {
+		return nil, nil, err
+	}
+
+	// Congestion Controller
+	bwEstimatorCh := make(chan cc.BandwidthEstimator, 1)
+	congestionController, err := cc.NewInterceptor(func() (cc.BandwidthEstimator, error) {
+		return gcc.NewSendSideBWE(
+			gcc.SendSideBWEInitialBitrate(int(float32(getRateForSimulcastLevel(SimulcastLevelLow))*0.50)),
```

Comment:
Originally I was starting with the medium quality track (we had three) mainly to avoid wasting too much time before upgrading to best quality (ideal case). But starting with high I think is a bit dangerous as it could cause bad congestion since at the start we don't know anything about the peer's network and can't asses its bandwidth. That said this is not set in stone, we can always review it later but I'd rather being on the safe side for now.